processes/release-process.md

```diff
@@ -123,3 +123,121 @@ versioned releases are available:
 - [{{repo}}](https://github.com/{{project.name}}/{{repo}})
 	{% endfor %}
 {% endfor %}
+
+## Release procedure
+
+This section describes how a release is done. Most of the points below are
+applicable to the wider public except for those where the actual release work
+is carried out. Note that this procedure is based on the Trustworthy Systems
+team-internal Bamboo continuous infrastructure and GitHub setup. This procedure
+is likely to change in the coming future as the Trustworthy Systems team
+migrates away from the Bamboo setup and perhaps experiments with a more
+frequent release cycle.
+
+### Release window and feeze
+
+When a release is planned, decide on the timeframe of the release window in
+which changes can still be made, and then make an announcement to the relevant
+channels that a release is coming up and let people know of the release window
+that you have decided. This is to give advance notice to contributors so that
+they can prepare for the release and get features merged in before the release
+is done. A decent timeframe for the release window is a month.
+
+The relevant channels for the announcement are:
+    - [seL4 discourse](https://sel4.discourse.group/)
```

Comment:
The list doesn't display correctly. You probably need a blank like before the first item, and no indent for the items.

Reply:
Oh woops, guess I skimmed over that and forgot to check that.